### ISSUE TITLE
Переименование папки с кешем

### ISSUE THREAD

**Issue by vivabelarus** [wrwrabbit/Partisan-Telegram-Android#119]
Возможность изменить название папки, в которой будет храниться кеш. В подпапках слово "Telegram", соответственно, тоже должно быть заменено. Можно, наверное, даже сделать раздельное хранение для разных аккаунтов.

**Comment by wrwrabbit** [comment]
Зачем это?

**Comment by vivabelarus** [comment]
Ну типа чтобы не знали, где искать. В комментах такое предложили. Если ещё совместить с шифрованием изображений, то было бы совсем неплохо. А шифрование изображений, кстати, есть, но только для секретных чатов, видимо. Но можно разобраться, можно ли перенести на обычные.

**Comment by vivabelarus** [comment]
Да, в секретных чатах вообще вполне неплохая защита, как оказалось. Странно, что на другие чаты не распространяется.

**Comment by Ultrafeel** [comment]
В свете статьи на хабре "Конфиденциальность пользователей Telegram снова нарушена. Представители мессенджера требуют не раскрывать подробностей", с этими кешами надо серьезней.
